Sources/PackageDescription/Target.swift

```diff
@@ -53,6 +53,22 @@ extension Target: TOMLConvertible {
     }
 }
 
+// MARK: StringLiteralConvertible
+
+extension Target.Dependency : StringLiteralConvertible{
```

Comment:
Add space between type and opening `}` bracket. Also use this style in other places.
Like this - `tringLiteralConvertible {`
Let's keep one common style. :)